puppetdb/Dockerfile

```diff
@@ -81,9 +81,11 @@ RUN apt update && \
 
 FROM base as release
 
+ARG PUPPET_VERSION
+ENV PUPPET_VERSION=${PUPPET_VERSION:-7}
```

Comment:
This works? Both ENV and ARG having same name?

Reply:
Yes this works: ARG gets the parameter from the outside, if any. Then it is put in its self as ENV, if it is empty, it will be "7". We also have this in the  PDC container and i have seen this in some other Dockerfiles like this.